src/lib/viewers/BaseViewer.js

```diff
@@ -994,6 +994,7 @@ class BaseViewer extends EventEmitter {
         const annotatorOptions = this.createAnnotatorOptions({
             annotator: this.annotatorConf,
             features: options && options.features,
+            initialColor: this.annotationModule.getColor(),
```

Comment:
In the future, we may want to consider supporting something like an `initial` object that gets merged into the annotator's initial state.